chess/4-database/database.md

```diff
@@ -42,6 +42,9 @@ As you design your database schema, carefully consider data types, primary and f
 > [!TIP]
 > The `DatabaseManager` class also has a method for creating a database if it does not exist. You are not required to use this code, but it is required that on start up, your code creates both your database and tables if they do not exist, based on the values configured in `db.properties`. This allows the pass off tests to run without manual intervention to set up your database.
 
+> [!TIP]
```

Comment:
This is what I am worried about with the overuse of "TIP", more tips then there is actual instruction. It is like using ALL CAPs in your DMs.